src/util/native_fs_utils.js

```diff
@@ -516,12 +516,18 @@ async function get_fs_context(nsfs_account_config, fs_backend) {
     if (nsfs_account_config.distinguished_name) {
         account_ids_by_dn = await get_user_by_distinguished_name({ distinguished_name: nsfs_account_config.distinguished_name });
     }
-    return {
+    const fs_context = {
         uid: (account_ids_by_dn && account_ids_by_dn.uid) ?? nsfs_account_config.uid,
         gid: (account_ids_by_dn && account_ids_by_dn.gid) ?? nsfs_account_config.gid,
         warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
-        backend: fs_backend
+        backend: fs_backend,
```

Comment:
You can remove the last comma
```suggestion
        backend: fs_backend
```